lib/verifyConditions.js

```diff
@@ -7,8 +7,8 @@ module.exports = async (pluginConfig, context) => {
 
     const env = context.env;
 
-    if (!pluginConfig.chartPath) {
-        errors.push('Missing argument: chartPath');
+    if (!pluginConfig.chartPath || !env.CHART_PATH) {
```

Comment:
You only have to set one of them

```suggestion
    if (!pluginConfig.chartPath && !env.CHART_PATH) {
```